client/post-editor/editor-post-formats/index.jsx

```diff
@@ -77,18 +90,20 @@ const EditorPostFormats = React.createClass( {
 	},
 
 	renderPostFormats: function() {
+		var selectedFormat = this.getSelectedPostFormat();
+
 		return this.getPostFormats().map( function( postFormat ) {
 			return (
 				<li key={ postFormat.slug } className="editor-post-formats__format">
 					<label>
 						<FormRadio
 							name="format"
 							value={ postFormat.slug }
-							checked={ postFormat.slug === this.props.value }
+							checked={ postFormat.slug === selectedFormat }
 							onChange={ this.onChange } />
 						<span className="editor-post-formats__format-label">
 							<span className={ 'editor-post-formats__format-icon' } >
-								<Gridicon icon={ this.getPostFormatIcon( postFormat ) } size={ 20 } />
+								<Gridicon icon={ this.getPostFormatIcon( postFormat ) } size={ 18 } />
```

Comment:
There was a lint failure on this line, that informed me the prior value of `20` did not follow proper Gridicon usage.  Not sure if we need to adjust any other styles because of this /cc @hugobaeta